cypress/support/pageObjects/inventoryPage.js

```diff
@@ -0,0 +1,88 @@
+var inventoryItem = require('../../models/inventoryItem.json');
+
+export class InventoryPage {
+    static wrapItems() {
+        let pageItems = [];
+        let amountOfItems = this.getAmountOfItems();
+        console.log('amountOfItems', amountOfItems);
+        for (let i = 1; i <= amountOfItems; i++) {
+            inventoryItem = {name: '', price: '', description: '', addToCartLocator: ''};
+            let itemJson = {...inventoryItem};
+            cy.get(`.inventory_list > :nth-child(${i})> .inventory_item_description>.inventory_item_label > a >.inventory_item_name`)
```

Comment:
Please think how you can simplify this locator

Reply:
And extract it to the method getItemName

Reply:
I moved locators to separate methods, but could not find a way to simplify them, because they are ordered (connected to their own number in the child list)

Reply:
I moved locators to separate methods, but could not find a way to simplify them, because they are ordered (connected to their own number in the child list)